package.json

```diff
@@ -115,7 +116,7 @@
     "stylelint-prettier": "^1.1.2",
     "stylelint-scss": "^3.18.0",
     "turbo": "^1.4.3",
-    "vite": "^4.5.2",
+    "vite": "^5.2.13",
```

Comment:
i don't think vite nor date-fns should be declared as dependencies at the root package.json of the monorepo

https://turbo.build/repo/docs/crafting-your-repository/managing-dependencies#install-dependencies-where-theyre-used

dependencies should be declared where they are used, the root package.json should ideally just contains the monorepo dependencies such as turbo / yarn / etc